tests/kani/Makefile

```diff
@@ -0,0 +1,38 @@
+SHELL=/bin/bash
+
+# Assume cbmc and cbmc-viewer are installed in PATH
+#   skip ./kani/scripts/setup/ubuntu/install_cbmc.sh
+#   skip ./kani/scripts/setup/install_viewer.sh 2.11
+
+KANI=kani
+
+# ARCH is ubuntu or macos-10.15
+ARCH=ubuntu
+
+clone:
+	git clone https://github.com/model-checking/kani.git $(KANI)
+	cd $(KANI) && git submodule update --init
+
+# MacOS warning: If brew fails trying to install ctags over
+# universal-ctags, then replace ctags with universal-ctags
+# ./kani/scripts/setup/macos-10.15/install_deps.sh
+
+build:
+	PATH=$$HOME/.cargo/bin:$$PATH && cd $(KANI) && \
+	    ./scripts/setup/$(ARCH)/install_deps.sh && \
+	    ./scripts/setup/install_rustup.sh && \
+	    cargo build --workspace
+
+test:
+	PATH=$$HOME/.cargo/bin:$$(pwd)/$(KANI)/scripts:$$PATH && \
+	echo cbmc --version: $$(cbmc --version) && \
```

Comment:
Make prints the commands out before running them, so this is going to print out
```
echo cbmc --version: $$(cbmc --version)
cbmc --version: 5.54
```
I suggest either `@echo` so that the commandline itself is not printed out. Or, simply write
```
cbmc --version
```
which would then print out
```
cbmc --version
5.54
```

Also, for diagnostic output, it's better to print to stderr, like
```
>&2 echo foo
```

Reply:
Done something similar.